WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample.Android/FirebaseService.cs

```diff
@@ -62,5 +72,25 @@ void SendMessageToMainPage(string body)
         {
             (App.Current.MainPage as MainPage)?.AddMessage(body);
         }
+
+        void SendRegistrationToServer(string token)
+        {
+            try
+            {
+                NotificationHub hub = new NotificationHub(AppConstants.NotificationHubName, AppConstants.ListenConnectionString, this);
+
+                // register device with Azure Notification Hub using the token from FCM
+                Registration reg = hub.Register(token, AppConstants.SubscriptionTags);
+
+                // subscribe to the SubscriptionTags list with a simple template.
+                string pnsHandle = reg.PNSHandle;
+                var cats = string.Join(", ", reg.Tags);
```

Comment:
What's the point of this? It's not used?

Reply:
This was copy/pasted from the class I deleted. I think this was the first way I passed tags before I created an App constant when I first wrote this sample. Removed.